docs/06_how-to-guides/40_multi-index/how-to-instantiate-a-multi-index-table.md

```diff
@@ -69,7 +69,7 @@ using namespace eosio;
 +  test_tables testtab;
 ```
 
-7. Instantiate the data member `testtab` by passing to its constructor the `scope` (in this case `receiver`) and the `code` parameters, these two combined with table name `"testtaba"` provide access to the partition of the RAM cache used by this multi index table, in this example you will initialize the `testtab` data member in the smart contract constructor
+7. Initialize the data member `testtab` by passing to its constructor for the `code` parameter the `receiver` value and for the `scope` parameter the `receiver.value` value. These two parameters combined with table name `"testtaba"` provide access to the partition of the RAM cache used by this multi-index table, in this example you will initialize the `testtab` data member in the smart contract constructor.
```

Comment:
The step 7 starting sentence sounds a bit off. How about this suggestion if it makes sense?

Initialize the data member `testab` by passing these values to its constructor - `receiver` value to the `code` parameter; and `receiver.value` to the `scope` parameter.

Reply:
> The table name is `testtab` right? I see multiple instances of `testtaba` in doc. Looks like a typo.

"testtaba" is the table name
testtab is a variable name

Reply:
> Also, is `ds` equivalent to `scope` parameter?
> 
> Because it says the `code` param gets a value of `receiver` and the `scope` param is assigned a value of `receiver.value`

nope, `ds` is on a different line of code, nothing to do with `testtab` initialization.